pkg/sfu/downtrack.go

```diff
@@ -387,6 +393,83 @@ func (d *DownTrack) WritePaddingRTP(bytesToSend int) int {
 	return bytesSent
 }
 
+func (d *DownTrack) WriteBlankFrameRTP() error {
+	// LK-TODO: Support other video codecs
+	if d.Kind() == webrtc.RTPCodecTypeAudio || d.mime != "video/vp8" {
+		return nil
+	}
+
+	// send a number of blank frames just in case there is loss.
+	// Intentionally ignoring check for mute or bandwidth constrained mute
+	// as this is used to clear client side buffer.
+	i := 0
+	for {
+		frameEndNeeded := false
+		if !d.munger.IsOnFrameBoundary() {
+			frameEndNeeded = true
+		}
+
+		sn, ts, err := d.munger.UpdateAndGetPaddingSnTs(frameEndNeeded)
+		if err != nil {
+			return err
+		}
+
+		adjustedTs := ts + uint32(i+1)*(d.codec.ClockRate/30) // assume 30 fps
+		if frameEndNeeded {
+			adjustedTs = ts
+		}
+		hdr := rtp.Header{
+			Version:        2,
+			Padding:        false,
+			Marker:         true,
+			PayloadType:    d.payloadType,
+			SequenceNumber: sn,
+			Timestamp:      adjustedTs,
+			SSRC:           d.ssrc,
+			CSRC:           []uint32{},
+		}
+
+		err = d.WriteRTPHeaderExtensions(&hdr)
+		if err != nil {
+			return err
+		}
+
+		blankVP8, err := d.vp8Munger.UpdateAndGetPadding(!frameEndNeeded)
+		if err != nil {
+			return err
+		}
+
+		// 1x1 key frame
+		// Used even when closing out a previous frame. Looks like receivers
+		// do not care about content (it will probably end up being an undecodable
+		// frame, but that should be okay as there are key frames following)
+		vp8PayloadHeader := []byte{0x10, 0x02, 0x00, 0x9d, 0x01, 0x2a, 0x01, 0x00, 0x01, 0x00, 0x0b, 0xc7, 0x08, 0x85, 0x85, 0x88, 0x85, 0x84, 0x88, 0x3f, 0x82, 0x00, 0x0c, 0x0d, 0x60, 0x00, 0xfe, 0xe6, 0xb5, 0x00}
```

Comment:
nit:  could we move this to a const?

Reply:
Done in this commit @davidzhao - https://github.com/livekit/livekit-server/pull/179/commits/d0af4b3d1e7aa926f1051889108bf65d8535bf63